### ISSUE TITLE
The list of bot commands and users when mentioning is not accessible without workarounds

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2191]
Steps:

1. In a chat press `/` or `@` , or press the bot commands button if there is any.
2. Press shift tab.
3. Navigate with arrows.

Expected: The list should read the user information or command information.

Actual: If the list contains bot commands, `Unigram.ViewModels.UserCommand` is read, else some json about the user is read. A workaround, of course, is to use object navigation, the containing object is has the relevant info.

**Comment by sukiletxe** [comment]
This has been fixed
